MobileBuy/sample/build.gradle

```diff
@@ -39,12 +39,14 @@ android {
       def apiKey
       def androidPayPublicKey
       def androidPayEnvironment
+      def defaultLocale
 
       // get the shop setting from the environment if there any
       shopDomain = System.getenv("SHOP_DOMAIN")
       apiKey = System.getenv("API_KEY")
       androidPayPublicKey = System.getenv("ANDROID_PAY_PUBLIC_KEY")
       androidPayEnvironment = System.getenv("ANDROID_PAY_ENVIRONMENT")
+      defaultLocale = System.getenv("DEFAULT_LOCALE") ?: ""
```

Comment:
nit: `locale` instead of `default_locale`? also default value can be `null` here as well.